mne/time_frequency/spectrum.py

```diff
@@ -652,10 +652,19 @@ def plot(
         scalings = _handle_default("scalings", None)
         titles = _handle_default("titles", None)
         units = _handle_default("units", None)
-        if amplitude == "auto":
+
+        depr_message = (
+            "The default value of `amplitude` will change from `amplitude='auto'` to "
+            "`amplitude=False` in MNE 1.8.0."
+        )
+        if amplitude is None or amplitude == "auto":
+            warn(depr_message, FutureWarning)
```

Comment:
the warning should say something like "auto is going away and the new default will be False". Otherwise it's not telling the whole picture, and it's also not useful for folks who set it to `"auto"` explicitly (they weren't relying on the default behavior anyway).